### ISSUE TITLE
Is tradeInfo object correct?

### ISSUE THREAD

**Issue by georgezouq** [tiagosiebler/TriangularArbitrage#52]
I check the structure of candidate object's attribute `a`, `b` and `c`. There's a `tradeInfo` object content `side, symbol, quantity`. 

<img width="517" alt="screen shot 2018-06-30 at 12 37 25 pm" src="https://user-images.githubusercontent.com/5990860/42121472-67f1cffe-7c62-11e8-962c-be4bc78b69f9.png">

I print it in TradeCore and I find

<img width="561" alt="screen shot 2018-06-30 at 12 25 31 pm" src="https://user-images.githubusercontent.com/5990860/42121395-cd518c42-7c60-11e8-981b-3e8c2cffd65e.png">

the `side` sames incorrect, If I trade by this, In the first, I sell `bnb` buy `eth`, then buy `qlc` sell `bnb` and then sell `qlc` buy `eth`, eventually, I use 2 time `bnb`, get 2 time `eth`(ignore quantity). I think after arbitrage, all trade done. I use 1 bnb should get maybe 1.1 bnb, that 0.1 bnb is profit. But trade by this, its not balance. Or each arbitrage candidate not alone？

**Comment by tiagosiebler** [comment]
You're correct, if I remember right I noticed this too in an internal branch of this experiment. Each step is a transaction, if you're going to BNB via BNBETH, you'd be buying BNB using ETH. PRs for enhancements & fixes are most welcome if you wish to work on it.
